src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --blue-color: #007bff;
```

Comment:
you use wrong blue color
```suggestion
  --blue-color: #00ACDC;
```
<img width="718" height="290" alt="image" src="https://github.com/user-attachments/assets/583d9793-4133-48a3-89a3-f63ac5443ba2" />